app/models/product_import/entry_validator.rb

```diff
@@ -282,6 +282,7 @@ def mark_as_existing_variant(entry, existing_variant)
     def product_field_errors(entry, existing_product)
       EntryValidator.non_updatable_fields.each do |display_name, attribute|
         next if attributes_match?(attribute, existing_product, entry) || attributes_blank?(attribute, existing_product, entry)
+        next if attribute == :description
```

Comment:
I think this requires a spec.

Reply:
This line would also be clearer if the condition is moved to a method with a more descriptive name, e.g. `ignore_attribute_when_updating_product?(attribute)`.

And then it is better to check this earlier in the block, so there is no need to compare values in `existing_product` and `entry` if the attribute should be skipped.